src/tests/ftest/dfuse/sparse_file.py

```diff
@@ -77,15 +77,15 @@ def test_sparsefile(self):
         fsize_after_truncate = get_remote_file_size(self.hostlist_clients[0], sparse_file)
         self.log.info("File size (in bytes) after truncate: %s", fsize_after_truncate)
         # verifying the file size got set to desired value
-        self.assertTrue(fsize_after_truncate == self.space_before)
+        self.assertEqual(fsize_after_truncate, self.space_before)
```

Comment:
Could we add a descriptive message here that will be displayed if the assert fails?

Reply:
I see where you're coming from but I'm not sure what this test is actually doing.  I might copy the comment above in as a message.

Reply:
From a quick look, maybe
"File size incorrect after truncate"